ntoskrnl/se/accesschk.c

```diff
@@ -922,17 +1369,20 @@ NtAccessCheck(
     {
         /* Now perform the access check */
         SepAccessCheck(CapturedSecurityDescriptor,
-                       &SubjectSecurityContext,
+                       Token,
+                       &SubjectSecurityContext.PrimaryToken,
+                       NULL,
                        DesiredAccess,
                        NULL,
                        0,
                        PreviouslyGrantedAccess,
-                       &PrivilegeSet, //FIXME
                        GenericMapping,
                        PreviousMode,
+                       FALSE,
+                       FALSE,
+                       &PrivilegeSet,
```

Comment:
The whole ordeal with this PrivilegeSet pointer is partially broken in this function. (Missing capture of the buffer, etc etc.)

Reply:
This `PrivilegeSet` is completely redundant as `NtAccessCheck` already allocates a privilege set buffer by the `SePrivilegePolicyCheck` call anyway. I'm going to remove `PrivilegeSet`from there.